lib/web/ui/integration.go

```diff
@@ -234,6 +234,29 @@ type AWSOIDCDeployDatabaseServiceResponse struct {
 	ClusterDashboardURL string `json:"clusterDashboardUrl"`
 }
 
+// AWSOIDCEnrollEKSClustersRequest is a request to ListEKSClusters using the AWS OIDC Integration.
+type AWSOIDCEnrollEKSClustersRequest struct {
+	Region             string   `json:"region"`
```

Comment:
As @marcoandredinis said, can we trim the params to the minimum required and get everything from backend including the token - i.e. we shouldn't share the token if it's not required to.

Reply:
Done.